State.lua

```diff
@@ -31,9 +31,16 @@ WarpDeplete.defaultState = {
 	deathDetails = {},
 
 	pullCount = 0,
+	extraCount = 0,
 	currentCount = 0,
 	totalCount = 100,
 
+	-- Keep track of what function has ran since all 3 run when a mob
+	-- worth force dies.
+	combatLogExecuted = false,
+	scenarioCriteriaExecuted = false,
+	scenarioPOIExecuted = false,
```

Comment:
Can you add some more details here on why we need to keep track of these? (I think I understand why, but future me might not :sweat_smile:)